common/src/main/java/com/ivy/common/Constants.kt

```diff
@@ -10,15 +10,15 @@ object Constants {
     const val FREE_LOANS = 2
 
     const val URL_TC =
-        "https://github.com/ILIYANGERMANOV/privacy-policies/blob/master/ivy-wallet-tc.md"
+        "https://github.com/Ivy-Apps/privacy-policies/blob/master/ivy-wallet-tc.md"
     const val URL_PRIVACY_POLICY =
```

Comment:
This won't work - revert for Privacy Policy aThis won't work - revert for Privacy Policy and T&C.nd T&C.